src/consts.py

```diff
@@ -28,3 +28,45 @@ class BITNESS(enum.Enum):
     CURRENT_BITNESS = BITNESS.B64
 else:
     CURRENT_BITNESS = BITNESS.B32
+
+COMA_SPLIT_BLACKLIST = [
```

Comment:
hate to be pedantic but it's spelled "COMMA" 😅

Reply:
No worries, it is important for reader aesthetic feelings ;)